tasks/bundle-platform

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
```

Comment:
Actually, it's more of a binary helper used by the tasks--I'll move it to `tasks/bin` if that makes sense. I just wanted it not to be in `chef-dk/bin` because, as @danielsdeleo pointed out, that would cause it to be distributed with the chef-dk :)